model-optimizer/extensions/front/CTCGreedyDecoderReplacement.py

```diff
@@ -89,3 +92,44 @@ def pattern(**kwargs):
 
     def replace_sub_graph(self, graph: Graph, match: dict):
         replace_ctc_greedy_decoder(graph, match)
+
+
+class CTCGreedyDecoderSingleReplacement(FrontReplacementSubgraph):
+    """
+    TensorFlow CTCGreedyDecoder produces output in a sparse tensor that is not supported by Inference Engine and
+    Inference Engine's CTCGreedyDecoderSeqLen has different output that is in a dense format. So this transformation
+    handle single TF CTCGreedyDecoder and warns user about another format of output.
+    """
+    enabled = True
+
+    @staticmethod
+    def pattern(**kwargs):
+        return dict(
+            nodes=[('decoder', dict(op='CTCGreedyDecoderSeqLen')),
+                   ('result', dict(op='Result')),
+                   ],
+            edges=[('decoder', 'result', {'out': 0})]
+        )
```

Comment:
I think this pattern is not needed. You need to walkthrough all `CTCGreedyDecoderSeqLen` and transform only one that have outputs ports going to `Result` node. You know that `CTCGreedyDecoderSeqLen` can have upto 3 outputs at this stage and you need to check all connected outputs.

Reply:
Yes, I update transform and remove pattern. But I cannot understand why operation may have upto 3 outputs, not 2.

Reply:
Discussed and resolved.